tests/Interface/core/ImageContainerTest.cpp

```diff
@@ -132,7 +164,25 @@ int main() {
   const Img<float, 2> testBracketOperator2(grayimage_bmp);
   // CHECK: 240.0
   fprintf(stderr, "%f\n", testBracketOperator2[15]);
-
+  //===--------------------------------------------------------------------===//
+  // Test Opencv Image without norm
+  //===--------------------------------------------------------------------===//
+  cv::Mat testImgcvbmp=cv::imread(
+		  "../../../../tests/Interface/core/TestGrayImage.bmp",
+		  cv::IMREAD_GRAYSCALE);
+  Img<float,2> testImgbmp(testImgcvbmp);
+  bool testbmp=testImgcvnorm<float,2>(testImgcvbmp,testImgbmp);
```

Comment:
Did you format this file with clang-format?

`git clang-format HEAD~` will format your entire previous commit in-place for you.